### ISSUE TITLE
Error: invalid character 'c' looking for beginning of value

### ISSUE THREAD

**Issue by cdorneles2433** [Azure/aztfexport#443]
Dear,

I am facing this error:

Error: parallel importing: 2 errors occurred:
	* task handler error: failed to merge state file: 2 errors occurred:
	* showing state file /tmp/aztfexport-4278575557/terraform.tfstate: invalid character 'c' looking for beginning of value
	* showing state file /tmp/2799109596/terraform.tfstate: exit status 1

	* task handler error: failed to merge state file: 2 errors occurred:
	* showing state file /tmp/aztfexport-4267937580/terraform.tfstate: invalid character 'c' looking for beginning of value
	* showing state file /tmp/4271385112/terraform.tfstate: exit status 1

when I tried to run these commands.:

*Map

**aztfexport resource-group --plain-ui --generate-mapping-file --non-interactive rg-xxxxxxx**

*Import

**aztfexport map --plain-ui --backend-type=azurerm --backend-config=resource_group_name=xxxxxx --backend-config=storage_account_name=xxxxxx --backend-config=container_name=xxxxxx --backend-config=key=xxxxx.tfstate --log-level TRACE -n "aztfexportResourceMapping.json"**

I am using the latest version of Ubuntu.

I am using these commands to install the tooling.

              `  sudo go install github.com/Azure/aztfexport@latest`
                `sudo apt-get install -y aztfexport`

Do you have any ideas? 

BR

Clairo

**Comment by magodo** [comment]
@cdorneles2433 Thank you for raising this issue! I've tried above workflow at my side, it all looks good. From the error message, it seems the error is raised from the call to `terraform show -json` against the state stored in your remote backend. Would you please try to run it directly against your remote backend?

**Comment by duzitong** [comment]
I met the same issue today and magically fixed it.

Steps in my workflow was:
  1. Setup go-lang
  2. Install `aztfexport@latest`
  3. Execute `aztfexport rg ...`

But I met no `terraform` executable found today which didn't occur last week. So I added one step before executing `aztfexport` to install Terraform by `setup-terraform@v2`.

Then, this issue occurs. I checked the terraform version installed is 1.5.6. I tested on my Windows machine with 1.5.6 Terraform, no problem raised.

Afterwards, I removed the terraform installation step. The terraform version is 1.5.5. This issue disappears.

I don't know why `aztfexport` is complaining missing `terraform` executable at the first place and why it's showing this error with setup Terraform action.
